src/ragas/utils.py

```diff
@@ -22,6 +25,9 @@
 }
 # endpoint for uploading results
 RAGAS_API_URL = "https://api.ragas.io"
+RAGAS_APP_URL = "https://app.ragas.io"
+RAGAS_API_VERSION = "1.0.0"
+RAGAS_API_SOURCE = "ragas_py"
```

Comment:
RAGAS_API_SOURCE should be hardcoded. Not picked up via env

Reply:
it is hard coded - its a constant